selectrum.el

```diff
@@ -691,7 +713,10 @@ select one of the listed candidates (so, for example,
       (let* ((minibuffer-allow-text-properties t)
              ;; Not currently supported as all of our state is global.
              (enable-recursive-minibuffers nil)
-             (selected (read-from-minibuffer prompt nil keymap nil t))
+             (minibuffer-history-variable
+              (or history 'minibuffer-history))
+             (selected (read-from-minibuffer
+                        prompt nil keymap nil minibuffer-history-variable))
```

Comment:
Why do we need to both bind `minibuffer-history-variable` and also pass the `HIST` argument?

Reply:
I initially did this so that functions running in `selectrum-candidate-selected-hook` have access to this info. The idea was that this way the user can manipulate the history in other ways if he wants.

Reply:
Does it work if we only bind `minibuffer-history-variable`, and don't pass the `HIST` argument? If so, I'd like to do that as it is cleaner.

Reply:
I have tested it and omitting the argument does not work. But I think binding state variables this way before the hook runs is still a good idea so the hook functions have more information about the completion.

Reply:
I think we can get away with passing nil to `read-from-minibuffer`.

Reply:
I tested again and this does not work for me, the variable is not picked up by `read-from-minibuffer` if nil is passed as history arg.

Reply:
Ok, fair enough. Not that important.